Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -86,7 +86,7 @@ extension BRBAccountViewController: UITableViewDataSource {
         let historyItem = account.history[indexPath.row]
         cell.configure(title: historyItem.name,
                        subtitle: historyItem.timestamp,
-                       amount: historyItem.amount)
+                       amount: historyItem.amount, positive: historyItem.positive)
```

Comment:
can we move `positive: historyItem.positive` onto its own line like the other arguments? i.e. it should look like
```
cell.configure(
  title: historyItem.name,
  subtitle: historyItem.timestamp,
  amount: historyItem.amount
  positive: historyItem.positive
)
```